sqlglot/parser.py

```diff
@@ -5361,6 +5361,15 @@ def _parse_function_call(
             alias = not known_function or upper in self.FUNCTIONS_WITH_ALIASED_ARGS
             args = self._parse_csv(lambda: self._parse_lambda(alias=alias))
 
+            post_func_comments = self._curr and self._curr.comments
+            if known_function and post_func_comments:
+                anonymous_meta = (exp.SQLGLOT_META, "anonymous")
+                # If the meta comment "/* sqlglot.meta anonymous */" is following the function
+                # call we'll construct it as exp.Anonymous, even if it's "known"
+                for comment in post_func_comments:
+                    if all(meta in comment.lower() for meta in anonymous_meta):
+                        known_function = False
```

Comment:
You wanna do `lower()` once, and also you can break after setting `known_function` in the branch, right?

Reply:
Yep sg